### PR TITLE
Pla 4772 saassy proxy

### PR DESCRIPTION
**Pull request by jereynolds**
Big change
 - strip out all state, webapp dependency, sns
 - split mitmproxy and devtime into separate containers
 - devtime publishes directly to aggregator queue
 - no team/token env variable configuration
 - remove proxy api since it has been moved to webapp

### CODE REVIEW

File: lib/middleware.js [hunk 1]
```diff
@@ -73,21 +75,21 @@ function headerJsonParser(req, res, next) {
...
 }
 
 function setup(app) {
-    app.use(sendImmediateProxyResponse);
-    app.use(bodyParser.urlencoded({ limit: '50mb', extended: true }));
-    app.use(headerJsonParser);
-    app.use(nestedBodyParser);
-    app.use(logRequest);
-    app.use(sendToHelper);
+    app.post('*', sendImmediateProxyResponse);
```
Comment (kimslava93) [inline lib/middleware.js]: why was it changed from middleware to simple requests?

Comment (jereynolds) [reply lib/middleware.js]: They are still middleware -- express sees that they take a `next` callback and places them in the middleware chain. However, we want to bind these only to POST requests because we are using GET for the /status route which should not use middleware

**Comment by jereynolds** [comment]
@kimslava93 the API got migrated into the webapp since the proxy will now be a hosted part of the SaaS infrastructure. This way the proxy is much lighter weight and we can make use of the CLI for some functionality

File: lib/logger.js [hunk 2]
```diff
@@ -11,5 +11,5 @@ function error(...args) {
 
 module.exports = {
     log,
-    error
+    error,
```
Comment (jereynolds) [inline lib/logger.js]: why did you add the comma?

**Comment by kimslava93** [comment]
If we are moving all those functions to webapp, it become too heavy. I think we have to move some stuff from webapp to separated microservices.

File: lib/logger.js (hunk 2 above)
Comment (kimslava93) [reply lib/logger.js]: Ooops. it was occasionally, I'll remove it.
